index.md

```diff
@@ -62,7 +62,7 @@ Our core activities are:
 - building a research software community at Utrecht University.
 
 We can help you with, for example:
-- defining technical implementation of your research plan or grant proposal;
+- defining technical implementation of your [research plan or grant proposal](_pages/Long-term-support/index.md);
```

Comment:
I don't think the link to the page is relevant here. I would expect info on the technical aspect of the grant proposal, the way it is now.